exercises/practice/clock/ClockTest.php

```diff
@@ -31,204 +9,527 @@ public static function setUpBeforeClass(): void
         require_once 'Clock.php';
     }
 
+    /**
+     * uuid a577bacc-106b-496e-9792-b3083ea8705e
+     * @testdox on the hour
+     */
     public function testOnTheHour(): void
     {
         $clock = new Clock(8);
 
-        $this->assertEquals('08:00', $clock->__toString());
+        $this->assertEquals('08:00', $clock);
```

Comment:
This is one of the magic tricks in PHP, but we currently have no concept (concept exercise) for students to learn this. So we should either explain the magic in the stub (for now) or explicitly call the method as done before:

```suggestion
        $this->assertEquals('08:00', $clock->__toString());
```

or

```suggestion
        $this->assertEquals('08:00', (string)$clock);
        // plus use `implements Stringable` in the students stub
        // plus a comment in the stub pointing to PHP docs on interface `Stringable`
```

@homersimpsons What do you think about this?

Reply:
If we want to show magic methods with this exercise, I think I would prefer the explicit `(string)` cast, it has the advantages to clearly show that this method is magic.

But I do not know if we want to show those here.

I believe we could have a concept to show `__toString`, `__set`, `__get` which I think are rather "common". `__construct` is a magic method but I believe it is already being taught.